taskmanager/features/bootstrap/FeatureContext.php

```diff
@@ -114,22 +157,49 @@ public function thereIsTaskNamedAssignedToUserNamed($name, User $user)
     }
 
     /**
-     * @Transform :task
+     * @Given there is a task named :taskName with :status status
      */
-    public function convertTaskNameToTask(string $task): Task
+    public function thereIsTaskNamedWithStatus(string $taskName, Status $status)
     {
-        $taskCollection = $this->taskRegistry->getByName($task);
+        $task = new Task($taskName, $status);
 
-        return reset($taskCollection);
+        $this->taskRegistry = new TaskRegistry();
```

Comment:
@mateuszmaslanka should we recreate `TaskRegistry` at this step? IMHO it should not be refreshed under this step